tokio-threadpool/src/lib.rs

```diff
@@ -596,12 +600,12 @@ impl ThreadPool {
     /// The handle is used to spawn futures onto the thread pool. It also
     /// implements the `Executor` trait.
     pub fn sender(&self) -> &Sender {
-        &self.inner
+        self.inner.as_ref().unwrap()
```

Comment:
tioli: `.expect("accessed after drop")` or something?

Reply:
Yeah, I'm not sure exactly what is idiomatic in this case. The unwrap triggering a panic is (should) unreachable. I definitely use`expect` when the panic is part of the public API. Not sure what is best here.